gradlew

```diff
@@ -1,164 +1,164 @@
-#!/usr/bin/env bash

```

Comment:
Alright. In general, I prefer to avoid adding generated files to git since they frequently change and aren't true source code. So my questions are:
1. What additional build commands are needed so we can remove this file from git and just have it regenerated during each build?
2. If there is a good enough reason to keep it in git then how often would this file likely change?